<!doctype html><!-- This site was created with Wowchemy. https://www.wowchemy.com --><!-- Last Published: February 22, 2023 --><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/vendor-bundle.min.16f785cdb553c8c4431db6775122af35.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/@fancyapps/fancybox@3.5.7/dist/jquery.fancybox.min.css integrity="sha512-H9jrZiiopUdsLpg94A333EfumgUBpO9MdbxStdeITo+KEIMaNfHNvwyjjDJb+ERPaRS6DpyRlKbvPUasNItRyw==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.dc93abdfc6ebac00d36cb8086608717e.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Dharmesh D Bhuva"><meta name=description content="A highly-customizable Hugo academic resume theme powered by Wowchemy website builder."><link rel=alternate hreflang=en-us href=https://bhuvad.github.io/><link rel=canonical href=https://bhuvad.github.io/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#4caf50"><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@wowchemy"><meta property="twitter:creator" content="@wowchemy"><meta property="twitter:image" content="https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_3.png"><meta property="og:site_name" content="bhuvad"><meta property="og:url" content="https://bhuvad.github.io/"><meta property="og:title" content="bhuvad"><meta property="og:description" content="A highly-customizable Hugo academic resume theme powered by Wowchemy website builder."><meta property="og:image" content="https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2022-10-15T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","potentialAction":{"@type":"SearchAction","target":"https://bhuvad.github.io/?q={search_term_string}","query-input":"required name=search_term_string"},"url":"https://bhuvad.github.io/"}</script><script src=https://identity.netlify.com/v1/netlify-identity-widget.js></script>
<link rel=alternate href=/index.xml type=application/rss+xml title=bhuvad><title>bhuvad</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main class=page-wrapper><script src=/js/wowchemy-init.min.b6a114893adb99eba63208f045d5db5e.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>bhuvad</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>bhuvad</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about data-target=#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#projects data-target=#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/#featured data-target=#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#softwares data-target=#softwares><span>Softwares</span></a></li><li class=nav-item><a class=nav-link href=/#talks data-target=#talks><span>Talks</span></a></li><li class=nav-item><a class=nav-link href=/#contact data-target=#contact><span>Contact</span></a></li><li class=nav-item><a class=nav-link href=/uploads/resume.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://twitter.com/bhuva_dd data-toggle=tooltip data-placement=bottom title="Follow me on Twitter" target=_blank rel=noopener aria-label="Follow me on Twitter"><i class="fab fa-twitter" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><span class="js-widget-page d-none"></span><section id=about class="home-section wg-about"><div class=home-section-bg></div><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class="avatar avatar-circle" width=270 height=270 src=/authors/admin/avatar_hu5d6b5e2e7a2f38c6bfdd740fce699233_88112_270x270_fill_q75_lanczos_center.jpg alt="Dharmesh D Bhuva"><div class=portrait-title><h2>Dharmesh D Bhuva</h2><h3>Senior post-doctoral researcher at SAiGENCI</h3><h3><a href=https://www.adelaide.edu.au/ target=_blank rel=noopener><span>The University of Adelaide</span></a></h3></div><ul class=network-icon aria-hidden=true><li><a href=/#contact aria-label=envelope><i class="fas fa-envelope big-icon"></i></a></li><li><a href=https://twitter.com/bhuva_dd target=_blank rel=noopener aria-label=twitter><i class="fab fa-twitter big-icon"></i></a></li><li><a href="https://scholar.google.com/citations?user=Jz-azvUAAAAJ&hl=en" target=_blank rel=noopener aria-label=google-scholar><i class="ai ai-google-scholar big-icon"></i></a></li><li><a href=https://github.com/bhuvad target=_blank rel=noopener aria-label=github><i class="fab fa-github big-icon"></i></a></li><li><a href=https://orcid.org/0000-0002-6398-9157 target=_blank rel=noopener aria-label=orcid><i class="ai ai-orcid big-icon"></i></a></li><li><a href=https://www.linkedin.com/in/dbhuva/ target=_blank rel=noopener aria-label=linkedin><i class="fab fa-linkedin big-icon"></i></a></li><li><a href=/uploads/resume.pdf aria-label=cv><i class="ai ai-cv big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-8"><h1>Biography</h1><div class=article-style><p>Dharmesh D Bhuva is an early career computational biology researcher (PhD in 2020) in the Davis laboratory at SAiGENCI. He has developed high quality bioinformatics software and methods that enable exploration of molecular phenotypes in cancer systems. He currently works on developing statistical and computational methods to analyse high throughput spatial transcriptomics datasets.</p><p><i class="fas fa-download pr-1 fa-fw"></i> Download my
<a href=/uploads/resume.pdf target=_blank>resumé</a>
.</p></div><div class=row><div class=col-md-5><div class=section-subheading>Interests</div><ul class="ul-interests mb-0"><li>Spatial Statistics</li><li>Cancer Systems Biology</li><li>Computational Biology</li></ul></div><div class=col-md-7><div class=section-subheading>Education</div><ul class="ul-edu fa-ul mb-0"><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>PhD in Mathematics and Statistics, 2020</p><p class=institution>The University of Melbourne</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>MSc in Bioinformatics, 2015</p><p class=institution>The University of Melbourne</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>BSc in Computer Science, 2013</p><p class=institution>The University of Southampton</p></div></li></ul></div></div></div></div></div></section><section id=experience class="home-section wg-experience"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Experience</h1></div><div class="col-12 col-lg-8"><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border exp-fill">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><a href=https://www.adelaide.edu.au/saigenci/ target=_blank rel=noopener><img src=/media/icons/brands/saigenci.svg width=56px height=56px alt="South Australian Immunogenomics Cancer Institute (SAiGENCI)" loading=lazy></a></div><div><div class="section-subheading card-title exp-title text-muted my-0">Senior Post-doctoral Researcher (Computational Systems Oncology)</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.adelaide.edu.au/saigenci/ target=_blank rel=noopener>South Australian Immunogenomics Cancer Institute (SAiGENCI)</a></div><div class="text-muted exp-meta">Feb 2023 –
Present
<span class=middot-divider></span>
<span>Adelaide</span></div></div></div><div class=card-text><p>Responsibilities include:</p><ul><li>Development of novel spatial statistics methods to analyse sub-cellular spatial ‘omics data.</li><li>Development of machine learning pipelines to automate analysis of histopathology images.</li><li>Student supervision</li></ul></div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><a href=https://www.wehi.edu.au/ target=_blank rel=noopener><img src=/media/icons/brands/WEHI_RGB_logo.svg width=56px height=56px alt=WEHI loading=lazy></a></div><div><div class="section-subheading card-title exp-title text-muted my-0">Post-doctoral Researcher (Bioinformatics and Biostatistics)</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.wehi.edu.au/ target=_blank rel=noopener>WEHI</a></div><div class="text-muted exp-meta">Jun 2020 –
Feb 2023
<span class=middot-divider></span>
<span>Melbourne</span></div></div></div><div class=card-text><p>Responsibilities include:</p><ul><li>Analysis of spatial and single-cell transcriptomics to identify mechanisms of drug-induced phenotypic plasticity in cancer.</li><li>Data analysis for commercial research collaboration between the CRC for Cancer Therapeutics and Pfizer Inc.</li><li>Analysis of generic and drug specific regulatory mechanisms in biological systems using data integration approaches.</li><li>Analysis of multi-omics data associated with drug response across a diverse set of biological models including cell lines and patients.</li></ul></div></div></div></div></div></div></div></div></section><section id=projects class="home-section wg-portfolio"><div class=home-section-bg></div><div class=container><div class="row justify-content-center"><div class="section-heading col-12 mb-3 text-center"><h1 class=mb-0>Projects</h1></div><div class=col-12><span class="d-none default-project-filter">*</span><div class=project-toolbar><div class=project-filters><div class="btn-toolbar d-flex justify-content-center"><div class="btn-group flex-wrap"><a href=# data-filter=* class="btn btn-primary btn-lg active">All</a>
<a href=# data-filter=.js-id-deep-learning class="btn btn-primary btn-lg">Deep Learning</a>
<a href=# data-filter=.js-id-demo class="btn btn-primary btn-lg">Other</a></div></div></div></div><div class="isotope projects-container row js-layout-row project-showcase"></div></div></div></div></section><section id=talks class="home-section wg-portfolio"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Recent & Upcoming Talks</h1></div><div class="col-12 col-lg-8"><span class="d-none default-project-filter">*</span><div class=project-toolbar><div class=project-filters><div class=btn-toolbar><div class="btn-group flex-wrap"><a href=# data-filter=* class="btn btn-primary btn-lg active">All</a>
<a href=# data-filter=.js-id-invited-talk class="btn btn-primary btn-lg">Invited talks</a>
<a href=# data-filter=.js-id-workshop class="btn btn-primary btn-lg">Workshops</a></div></div></div></div><div class="isotope projects-container row js-layout-row"><div class="col-12 isotope-item js-id-invited-talk js-id-gene-set-enrichment-analysis js-id-molecular-phenotypes js-id-networks js-id-breast-cancer js-id-emt js-id-workshop"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/network-analysis-of-differentially-expressed-genes-using-visse-and-cytoscape/>Network analysis of differentially-expressed genes using vissE and Cytoscape</a></div><a href=/talk/network-analysis-of-differentially-expressed-genes-using-visse-and-cytoscape/ class=summary-link><div class=article-style>This workshop presents our vissE tool and shows how protein-protein interaction networks derived from it can be further interpreted using the Cytoscape network analysis tool.</div></a><div class="stream-meta article-metadata"><div><span>Sep 29, 2022</span>
<span class=middot-divider></span>
<span>Melbourne Bioinformatics</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Ahmed Mohamed</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-invited-talk js-id-networks js-id-regulatory-networks js-id-differential-co-expression js-id-breast-cancer"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/evaluation-and-characterisation-of-differential-co-expression-analysis-methods/>Evaluation and characterisation of differential co-expression analysis methods</a></div><a href=/talk/evaluation-and-characterisation-of-differential-co-expression-analysis-methods/ class=summary-link><div class=article-style>This talk presents a simple mathematical model of gene regulation used to simulate expression data to evaluate differential co-expression network inference methods.</div></a><div class="stream-meta article-metadata"><div><span>May 25, 2022 12:00 AM</span>
<span class=middot-divider></span>
<span>Online</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-invited-talk js-id-gene-set-enrichment-analysis js-id-molecular-phenotypes js-id-networks js-id-breast-cancer js-id-emt js-id-workshop"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/network-based-functional-analysis-in-transcriptomics/>Network-based functional analysis in transcriptomics</a></div><a href=/talk/network-based-functional-analysis-in-transcriptomics/ class=summary-link><div class=article-style>A workshop on using vissE to analyse transcriptomic data</div></a><div class="stream-meta article-metadata"><div><span>Nov 30, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-gene-set-enrichment-analysis js-id-molecular-phenotypes js-id-networks js-id-breast-cancer js-id-emt"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/visse-visualising-higher-order-biological-processes-in-cancer-omics-datasets/>vissE – Visualising higher-order biological processes in cancer ‘omics datasets</a></div><a href=/talk/visse-visualising-higher-order-biological-processes-in-cancer-omics-datasets/ class=summary-link><div class=article-style>A talk on using vissE to analyse transcriptomic data</div></a><div class="stream-meta article-metadata"><div><span>Oct 8, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-invited-talk js-id-gene-set-enrichment-analysis js-id-molecular-phenotypes js-id-networks js-id-breast-cancer js-id-emt js-id-workshop"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/an-introduction-to-biological-network-analysis/>An introduction to biological network analysis</a></div><a href=/talk/an-introduction-to-biological-network-analysis/ class=summary-link><div class=article-style>This masterclass includes an introductory lecture on gene-set enrichment analysis followed by a demonstration of the vissE and singscore tools we developed.</div></a><div class="stream-meta article-metadata"><div><span>Aug 27, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online (South-east Asia region)</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Jinjin Chen</span>, <span>Yimin Zhang</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-gene-set-enrichment-analysis js-id-molecular-phenotypes js-id-networks js-id-breast-cancer js-id-emt"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/visse-visualising-geneset-enrichment-analysis-results-using-a-network-based-approach/>vissE – Visualising gene–set enrichment analysis results using a network-based approach</a></div><a href=/talk/visse-visualising-geneset-enrichment-analysis-results-using-a-network-based-approach/ class=summary-link><div class=article-style>A talk on the inner workings of vissE</div></a><div class="stream-meta article-metadata"><div><span>Jul 27, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-invited-talk js-id-gene-set-enrichment-analysis js-id-molecular-phenotypes js-id-breast-cancer js-id-emt js-id-workshop"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/single-sample-gene-set-testing-of-tcga-rna-seq-data-workshop/>Single-Sample Gene-Set Testing of TCGA RNA-Seq Data Workshop</a></div><a href=/talk/single-sample-gene-set-testing-of-tcga-rna-seq-data-workshop/ class=summary-link><div class=article-style>This workshop demonstrates a workflow for single-sample gene-set enrichment analysis.</div></a><div class="stream-meta article-metadata"><div><span>Dec 13, 2020 12:00 AM</span>
<span class=middot-divider></span>
<span>Online (Nanjing University, China)</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Jinjin Chen</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div></div></div></div></div></div></section><section id=featured class="home-section wg-collection"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Featured Publications</h1></div><div class="col-12 col-lg-8"><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Ning Liu</span>, <span>Holly J Whitfield</span>, <span>Nicholas Papachristos</span>, <span>Sam Lee</span>, <span>Malvika Kharbanda</span>, <span>Ahmed Mohamed</span>, <span>Melissa J Davis</span></div><span class=article-date>March, 2022</span>
<span class=middot-divider></span>
<span class=pub-publication></span></div><a href=/publication/2022_bhuva_visse/><div class=img-hover-zoom><img src=/publication/2022_bhuva_visse/featured_hu41b3d5fe265c16b05a9161fd859c3f8a_857746_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2022_bhuva_visse/>vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis</a></div><a href=/publication/2022_bhuva_visse/ class=summary-link><div class=article-style><p>vissE is a tool that can help identify, visualise and explore higher-order biological processes from the results of a gene-set enrichment analysis (GSEA)</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.biorxiv.org/content/10.1101/2022.03.06.483195v1.full.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_bhuva_visse/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1101/2022.03.06.483195 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://vissE.cloud target=_blank rel=noopener>Web tool</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/vissE.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow/ target=_blank rel=noopener>Workshop</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span>Momeneh Foroutan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Ruqian Lyu</span>, <span>Kristy Horan</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>November, 2018</span>
<span class=middot-divider></span>
<span class=pub-publication>BMC Bioinform</span></div><a href=/publication/2018_foroutan_singscore/><div class=img-hover-zoom><img src=/publication/2018_foroutan_singscore/featured_hub771a9a426745829c866a5131a2ff602_79933_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Single sample scoring of molecular phenotypes" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2018_foroutan_singscore/>Single sample scoring of molecular phenotypes</a></div><a href=/publication/2018_foroutan_singscore/ class=summary-link><div class=article-style><p>singscore is a robust rank-based method to compute molecular phenotypes scores for individual samples such as those originating from an individual patient in the clinic</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bmcbioinformatics.biomedcentral.com/counter/pdf/10.1186/s12859-018-2435-4.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2018_foroutan_singscore/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.youtube.com/watch?v=pyHG9Qww-xk" target=_blank rel=noopener>Video</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1186/s12859-018-2435-4 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/singscore.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a></div></div></div></div></div></section><section id=publications class="home-section wg-collection"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Recent Publications</h1></div><div class="col-12 col-lg-8"><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Arutha Kulasinghe</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>NingLiu</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Chin WeeTan</span>, <span>James Monkman</span>, <span>Jane E Sinclair</span>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>David Godbolt</span>, <span>Liuliu Pan</span>, <span>Andy Nam</span>, <span>Habib Sadeghirad</span>, <span>Kei Sato</span>, <span>Gianluigi Li Bassi</span>, <span>Ken O'Byrne</span>, <span>Camila Hartmann</span>, <span>Anna Flavia Ribeiro dos Santos Miggiolaro</span>, <span>Gustavo Lenci Marques</span>, <span>Lidia ZytynskiMoura</span>, <span>Derek Richard</span>, <span>Mark Adams</span>, <span>Lucia de Noronha</span>, <span>Cristina Pellegrino Baena</span>, <span>Jacky Y Suen</span>, <span>Rakesh Arora</span>, <span>Gabrielle T Belz</span>, <span>Kirsty R Short</span>, <span>Melissa J Davis</span>, <span>Fernando Souza-Fonseca Guimaraes</span>, <span>John F Fraser</span></span>
(2022).
<a href=/publication/2022_kulasinghe_covidheart/>Transcriptomic profiling of cardiac tissues from SARS-CoV-2 patients identifies DNA damage</a>.
Immunology.<p><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_kulasinghe_covidheart/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/standR target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1111/imm.13577 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/standR.html target=_blank rel=noopener>Bioconductor</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Arutha Kulasinghe</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Chin Wee Tan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Anna Flavia Ribeiro dos Santos Miggiolaro</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>James Monkman</span>, <span>Habib Sadeghirad</span>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Jarbas da Silva Motta Junior</span>, <span>Caroline Busatta Vaz de Paula</span>, <span>Seigo Nagashima</span>, <span>Cristina Pellegrino Baena</span>, <span>Paulo Souza-Fonseca-Guimaraes</span>, <span>Lucia de Noronha</span>, <span>Timothy McCulloch</span>, <span>Gustavo Rodrigues Rossi</span>, <span>Caroline Cooper</span>, <span>Benjamin Tang</span>, <span>Kirsty R Short</span>, <span>Melissa J Davis</span>, <span>Fernando Souza-Fonseca Guimaraes</span>, <span>Gabrielle T Belz</span>, <span>Ken O'Byrne</span></span>
(2022).
<a href=/publication/2022_kulasinghe_covidlung/>Profiling of lung SARS-CoV-2 and influenza virus infection dissects virus-specific host responses and gene signatures</a>.
ERJ.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://erj.ersjournals.com/content/erj/59/6/2101881.full.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_kulasinghe_covidlung/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/standR target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1183/13993003.01881-2021 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/standR.html target=_blank rel=noopener>Bioconductor</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Jean Berthelet</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Momeneh Foroutan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Holly J Whitfield</span>, <span>Farrah El-Saafin</span>, <span>Joseph Cursons</span>, <span>Antonin Serrano</span>, <span>Michal Merdas</span>, <span>Elgene Lim</span>, <span>Emmanuelle Charafe-Jauffret</span>, <span>Christophe Ginestier</span>, <span>Matthias Ernst</span>, <span>Frédéric Hollande</span>, <span>Robin L Anderson</span>, <span>Bhupinder Pal</span>, <span>Belinda Yeo</span>, <span>Melissa J Davis</span>, <span>Delphine Merino</span></span>
(2022).
<a href=/publication/2022_berthelet_drugpred/>Computational Screening of Anti-Cancer Drugs Identifies a New BRCA Independent Gene Expression Signature to Predict Breast Cancer Sensitivity to Cisplatin</a>.
Cancers.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.mdpi.com/2072-6694/14/10/2404/pdf?version=1652425461" target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_berthelet_drugpred/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.3390/cancers14102404 target=_blank rel=noopener>DOI</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Ning Liu</span>, <span>Holly J Whitfield</span>, <span>Nicholas Papachristos</span>, <span>Sam Lee</span>, <span>Malvika Kharbanda</span>, <span>Ahmed Mohamed</span>, <span>Melissa J Davis</span></span>
(2022).
<a href=/publication/2022_bhuva_visse/>vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis</a>.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.biorxiv.org/content/10.1101/2022.03.06.483195v1.full.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_bhuva_visse/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1101/2022.03.06.483195 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://vissE.cloud target=_blank rel=noopener>Web tool</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/vissE.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow/ target=_blank rel=noopener>Workshop</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Laura Genovesi</span>, <span>Amanda Millar</span>, <span>Elissa Tolson</span>, <span>Matthew Singleton</span>, <span>Emily Hassall</span>, <span>Marija Kojic</span>, <span>Caterina Brighi</span>, <span>Emily Girard</span>, <span>Clara Andradas</span>, <span>Mani Kuchibhotla</span>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Raelene Endersby</span>, <span>Nicholas G Gottardo</span>, <span>Anne Bernard</span>, <span>Christelle Adolphe</span>, <span>James M Olson</span>, <span>Melissa J Davis</span>, <span>Brandon J Wainwright</span></span>
(2021).
<a href=/publication/2021_genovesi_mb/>Systems pharmacogenomics identifies novel targets and clinically actionable therapeutics for medulloblastoma</a>.
Genome Med.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://genomemedicine.biomedcentral.com/counter/pdf/10.1186/s13073-021-00920-z.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2021_genovesi_mb/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1186/s13073-021-00920-z target=_blank rel=noopener>DOI</a></p></div><div class=see-all><a href=/publication/>See all publications
<i class="fas fa-angle-right"></i></a></div></div></div></div></section><section id=softwares class="home-section wg-collection"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Recent softwares</h1></div><div class="col-12 col-lg-8"><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span></div><span class=article-date>Oct 15, 2022</span></div><a href=/softwares/dcanr/><div class=img-hover-zoom><img src=/softwares/dcanr/featured_hu86d665dad16700006f56a8957fd529b1_656008_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt=dcanr loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/dcanr/>dcanr</a></div><a href=/softwares/dcanr/ class=summary-link><div class=article-style><p>This package implements methods and an evaluation framework to infer differential co-expression/association networks. Various methods are implemented and can be evaluated using simulated datasets. Inference of differential co-expression networks can allow identification of networks that are altered between two conditions (e.g., health and disease).</p></div></a><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/dcanr/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/dcanr target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.dcanr target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/dcanr/ target=_blank rel=noopener>Webpage</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/dcanr/issues target=_blank rel=noopener>Report a bug</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span>Malvika D Kharbanda</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Chin Wee Tan</span>, <span class=author-highlighted>Dharmesh D Bhuva</span></div><span class=article-date>Oct 15, 2022</span></div><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/emtdata/>emtdata</a></div><a href=/softwares/emtdata/ class=summary-link><div class=article-style><p>This package provides pre-processed RNA-seq data where the epithelial to mesenchymal transition was induced on cell lines. These data come from three publications Cursons et al. (2015), Cursons et al. (2018) and Foroutan et al. (2017). In each of these publications, EMT was induces across multiple cell lines following treatment by TGFb among other stimulants. This data will be useful in determining the regulatory programs modified in order to achieve an EMT. Data were processed by the Davis laboratory in the Bioinformatics division at WEHI.</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/emtdata target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.emtdata target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/emtdata/issues target=_blank rel=noopener>Report a bug</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Gordon K Smyth</span>, <span>Alexandra Garnham</span></div><span class=article-date>Oct 15, 2022</span></div><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/msigdb/>msigdb</a></div><a href=/softwares/msigdb/ class=summary-link><div class=article-style><p>This package provides the Molecular Signatures Database (MSigDB) in a R accessible objects. Signatures are stored in GeneSet class objects form the GSEABase package and the entire database is stored in a GeneSetCollection object. These data are then hosted on the ExperimentHub. Data used in this package was obtained from the MSigDB of the Broad Institute. Metadata for each gene set is stored along with the gene set in the GeneSet class object.</p></div></a><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/msigdb/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/msigdb target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.msigdb target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/msigdb/issues target=_blank rel=noopener>Report a bug</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span>Ruqian Lyu</span>, <span>Momeneh Foroutan</span>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Malvika Kharbanda</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i></div><span class=article-date>Oct 15, 2022</span></div><a href=/softwares/singscore/><div class=img-hover-zoom><img src=/softwares/singscore/featured_hu7b79064d40f6ec3cd5c6251a5155e61b_251508_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt=singscore loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/singscore/>singscore</a></div><a href=/softwares/singscore/ class=summary-link><div class=article-style><p>A simple single-sample gene signature scoring method that uses rank-based statistics to analyze the sample&rsquo;s gene expression profile. It scores the expression activities of gene sets at a single-sample level.</p></div></a><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/singscore/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.singscore target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/singscore/ target=_blank rel=noopener>Webpage</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore/issues target=_blank rel=noopener>Report a bug</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Momeneh Foroutan</span>, <span>Yi Xie</span>, <span>Ruqian Lyu</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>Oct 15, 2022</span></div><a href=/softwares/singscoreamlmutations/><div class=img-hover-zoom><img src=/softwares/singscoreamlmutations/featured_huab255f603220787dd7053be139cd5651_93635_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt=SingscoreAMLMutations loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/singscoreamlmutations/>SingscoreAMLMutations</a></div><a href=/softwares/singscoreamlmutations/ class=summary-link><div class=article-style><p>This workflow package shows how transcriptomic signatures can be used to infer phenotypes. The workflow begins by showing how the TCGA AML transcriptomic data can be downloaded and processed using the TCGAbiolinks packages. It then shows how samples can be scored using the singscore package and signatures from the MSigDB. Finally, the predictive capacity of scores in the context of predicting a specific mutation in AML is shown.The workflow exhibits the interplay of Bioconductor packages to achieve a gene-set level analysis.</p></div></a><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/singscoreamlmutations/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/SingscoreAMLMutations target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.SingscoreAMLMutations target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/SingscoreAMLMutations/issues target=_blank rel=noopener>Report a bug</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span>Ning Liu</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Ahmed Mohamed</span></div><span class=article-date>Oct 15, 2022</span></div><a href=/softwares/standr/><div class=img-hover-zoom><img src=/softwares/standr/featured_hue1718606f0a0abe99ae5f5c9bb4c67c9_527212_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt=standR loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/standr/>standR</a></div><a href=/softwares/standr/ class=summary-link><div class=article-style><p>standR is an user-friendly R package providing functions to assist conducting good-practice analysis of Nanostring&rsquo;s GeoMX DSP data. All functions in the package are built based on the SpatialExperiment object, allowing integration into various spatial transcriptomics-related packages from Bioconductor. standR allows data inspection, quality control, normalization, batch correction and evaluation with informative visualizations.</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/standR target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.standR target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/standR target=_blank rel=noopener>Webpage</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GeoMXAnalysisWorkflow target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/standR/issues target=_blank rel=noopener>Report a bug</a></div></div><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Ahmed Mohamed</span></div><span class=article-date>Oct 15, 2022</span></div><a href=/softwares/visse/><div class=img-hover-zoom><img src=/softwares/visse/featured_hubb7914728c178d6e3f9604d4a552d94b_888199_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt=vissE loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/softwares/visse/>vissE</a></div><a href=/softwares/visse/ class=summary-link><div class=article-style><p>This package enables the interpretation and analysis of results from a gene set enrichment analysis using network-based and text-mining approaches. Most enrichment analyses result in large lists of significant gene sets that are difficult to interpret. Tools in this package help build a similarity-based network of significant gene sets from a gene set enrichment analysis that can then be investigated for their biological function using text-mining approaches.</p></div></a><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/visse/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.vissE target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/vissE/ target=_blank rel=noopener>Webpage</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE/issues target=_blank rel=noopener>Report a bug</a></div></div></div></div></div></section><section id=tags class="home-section wg-tag-cloud"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Popular Topics</h1></div><div class="col-12 col-lg-8"><div class=tag-cloud><a href=/tag/academic/ style=font-size:.7rem>Academic</a>
<a href=/tag/aml/ style=font-size:.7rem>AML</a>
<a href=/tag/brain-cancer/ style=font-size:.7rem>brain cancer</a>
<a href=/tag/breast-cancer/ style=font-size:1.8401819250574247rem>breast cancer</a>
<a href=/tag/covid/ style=font-size:1.0180457047536937rem>COVID</a>
<a href=/tag/differential-co-expression/ style=font-size:1.204090515550037rem>differential co-expression</a>
<a href=/tag/drug-reponse-prediction/ style=font-size:.7rem>drug reponse prediction</a>
<a href=/tag/emt/ style=font-size:1.7081810311000742rem>EMT</a>
<a href=/tag/endogenous-control-rna/ style=font-size:.7rem>endogenous control RNA</a>
<a href=/tag/gene-set-enrichment-analysis/ style=font-size:1.9721828190147752rem>gene-set enrichment analysis</a>
<a href=/tag/invited-talk/ style=font-size:1.4384792573258527rem>invited talk</a>
<a href=/tag/molecular-phenotypes/ style=font-size:1.94256977287589rem>molecular phenotypes</a>
<a href=/tag/networks/ style=font-size:1.800257367196711rem>networks</a>
<a href=/tag/pharmacogenomics/ style=font-size:1.0180457047536937rem>pharmacogenomics</a>
<a href=/tag/predict-mutation-status/ style=font-size:1.0180457047536937rem>predict mutation status</a>
<a href=/tag/r/bioconductor/ style=font-size:1.592867174679562rem>R/Bioconductor</a>
<a href=/tag/regulatory-networks/ style=font-size:1.204090515550037rem>regulatory networks</a>
<a href=/tag/spatial-transcriptomics/ style=font-size:1.204090515550037rem>spatial transcriptomics</a>
<a href=/tag/stably-expressed-genes/ style=font-size:.7rem>stably expressed genes</a>
<a href=/tag/workshop/ style=font-size:1.3360914095073877rem>workshop</a></div></div></div></div></section><section id=contact class="home-section wg-contact"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Contact</h1></div><div class="col-12 col-lg-8"><div class=mb-3><form name=contact method=post netlify netlify-honeypot=_gotcha><div class="form-group form-inline"><label class=sr-only for=inputName>Name</label>
<input type=text name=name class="form-control w-100" id=inputName placeholder=Name required></div><div class="form-group form-inline"><label class=sr-only for=inputEmail>Email</label>
<input type=email name=email class="form-control w-100" id=inputEmail placeholder=Email required></div><div class=form-group><label class=sr-only for=inputMessage>Message</label>
<textarea name=message class=form-control id=inputMessage rows=5 placeholder=Message required></textarea></div><div class=d-none><label>Do not fill this field unless you are a bot: <input name=_gotcha></label></div><button type=submit class="btn btn-primary px-3 py-2 w-100">Send</button></form></div><ul class=fa-ul><li><i class="fa-li fas fa-envelope fa-2x" aria-hidden=true></i>
<span id=person-email><a href=mailto:test@example.org>test@example.org</a></span></li><li><i class="fa-li fas fa-phone fa-2x" aria-hidden=true></i>
<span id=person-telephone><a href=tel:888%20888%2088%2088>888 888 88 88</a></span></li><li><i class="fa-li fas fa-map-marker fa-2x" aria-hidden=true></i>
<span id=person-address>450 Serra Mall, Stanford, CA 94305</span></li><li><i class="fa-li fas fa-compass fa-2x" aria-hidden=true></i>
<span>Enter Building 1 and take the stairs to Office 200 on Floor 2</span></li><li><i class="fa-li fas fa-clock fa-2x" aria-hidden=true></i>
<span>Monday 10:00 to 13:00<br>Wednesday 09:00 to 10:00</span></li><li><i class="fa-li fas fa-calendar-check fa-2x" aria-hidden=true></i>
<a href=https://calendly.com target=_blank rel=noopener>Book an appointment</a></li><li><i class="fa-li fab fa-twitter fa-2x" aria-hidden=true></i>
<a href=https://twitter.com/Twitter target=_blank rel=noopener>DM Me</a></li><li><i class="fa-li fas fa-video fa-2x" aria-hidden=true></i>
<a href=https://zoom.com target=_blank rel=noopener>Zoom Me</a></li></ul></div></div></div></section></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2023, Dharmesh D Bhuva. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.d26509351aa0ff874abbee824e982e9b.js></script>
<script src=https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/npm/@fancyapps/fancybox@3.5.7/dist/jquery.fancybox.min.js integrity="sha512-uURl+ZXMBrF4AwGaWmEetzrd+J5/8NRkWAvJx5sbPSSuOb0bZLqf+tOzniObO00BjHa/dD7gub9oCGMLPQHtQA==" crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":false}</script><script src=/en/js/wowchemy.min.e8ee06ba8371980ffde659871dd593b0.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>