---
title: An introduction to biological network analysis

event: 'WEHI Bioinformatics Masterclass 2021'
event_url: ''

location: Online (South-east Asia region)

summary: This masterclass includes an introductory lecture on gene-set enrichment analysis followed by a demonstration of the vissE and singscore tools we developed.
abstract: 'This workshop will focus on performing gene-set enrichment analysis of transcriptomic data and visualising the results of enrichment analysis. We will perform single-sample gene-set enrichment using methods in the singscore package to explore molecular phenotypes in individual samples. Following this, we will perform gene-set enrichment analysis using tools from the limma and edgeR packages. Finally, we will demonstrate a graph-based approach to visualise, summarise and interpret resutls of gene-set enrichment analysis. The workshop will be organised into two broad sections: 1) Molecular phenotyping of individual samples, and 2) Identifying and visualising higher-order phenotypes'

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: '2021-08-27'
#date_end: ''
all_day: false

# Schedule page publish date (NOT talk date).
publishDate: '2021-08-27'

authors:
  - admin
  - Chin Wee Tan
  - Jinjin Chen
  - Yimin Zhang
tags:
  - invited talk
  - gene-set enrichment analysis
  - molecular phenotypes
  - networks
  - breast cancer
  - EMT

# Is this a featured talk? (true/false)
featured: false

image:
  caption: ''
  focal_point: Right

links:
  - icon: twitter
    icon_pack: fab
    name: Follow
    url: https://twitter.com/bhuva_dd
url_code: ''
url_pdf: ''
url_slides: ''
url_video: ''

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ''

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
---
