---
title: "vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis"
authors:
- admin
- Chin Wee Tan
- Ning Liu
- Holly J Whitfield
- Nicholas Papachristos
- Sam Lee
- Malvika Kharbanda
- Ahmed Mohamed
- Melissa J Davis
date: "2022-03-07"
doi: "https://doi.org/10.1101/2022.03.06.483195"

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: ""
publication_short: ""

abstract: "Functional analysis of high throughput experiments using pathway analysis is now ubiquitous. Though powerful, these methods often produce thousands of redundant results owing to knowledgebase redundancies upstream. This scale of results hinders extensive exploration by biologists and often leads to investigator biases due to previous knowledge and expectations. To address this issue, we present vissE, a flexible network-based analysis method that summarises redundancies into biological themes and provides various analytical modules to characterise and visualise them with respect to the underlying data, thus providing a comprehensive view of the biological system. We demonstrate vissE's versatility by applying it to three different technologies: bulk, single-cell and spatial transcriptomics. Applying vissE to a factor analysis of a breast cancer spatial transcriptomic data, we identified stromal phenotypes that support tumour dissemination. Its adaptability allows vissE to enhance all existing gene-set enrichment and pathway analysis workflows, removing investigator bias from molecular discovery."

# Summary. An optional shortened abstract.
summary: vissE is a tool that can help identify, visualise and explore higher-order biological processes from the results of a gene-set enrichment analysis (GSEA)

tags:
- gene-set enrichment analysis
- molecular phenotypes
- networks
- breast cancer
- EMT
- spatial
- single-cell
featured: true

links:
- name: Web tool
  url: https://vissE.cloud
- name: Bioconductor
  url: https://www.bioconductor.org/packages/release/bioc/html/vissE.html
- name: Workshop
  url: https://davislaboratory.github.io/GenesetAnalysisWorkflow/
url_pdf: 'https://www.biorxiv.org/content/10.1101/2022.03.06.483195v1.full.pdf'
url_code: 'https://github.com/DavisLaboratory/vissE'
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Schematic of the vissE workflow'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
- internal-project

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
