<!doctype html><!-- This site was created with Wowchemy. https://www.wowchemy.com --><!-- Last Published: February 22, 2023 --><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/vendor-bundle.min.16f785cdb553c8c4431db6775122af35.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.dc93abdfc6ebac00d36cb8086608717e.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Dharmesh D Bhuva"><meta name=description content="A highly-customizable Hugo academic resume theme powered by Wowchemy website builder."><link rel=alternate hreflang=en-us href=https://bhuvad.github.io/tag/molecular-phenotypes/><link rel=canonical href=https://bhuvad.github.io/tag/molecular-phenotypes/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#4caf50"><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@wowchemy"><meta property="twitter:creator" content="@wowchemy"><meta property="twitter:image" content="https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_3.png"><meta property="og:site_name" content="bhuvad"><meta property="og:url" content="https://bhuvad.github.io/tag/molecular-phenotypes/"><meta property="og:title" content="molecular phenotypes | bhuvad"><meta property="og:description" content="A highly-customizable Hugo academic resume theme powered by Wowchemy website builder."><meta property="og:image" content="https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2022-10-15T00:00:00+00:00"><link rel=alternate href=/tag/molecular-phenotypes/index.xml type=application/rss+xml title=bhuvad><title>molecular phenotypes | bhuvad</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper><script src=/js/wowchemy-init.min.b6a114893adb99eba63208f045d5db5e.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>bhuvad</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>bhuvad</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#softwares><span>Softwares</span></a></li><li class=nav-item><a class=nav-link href=/#talks><span>Talks</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li><li class=nav-item><a class=nav-link href=/uploads/resume.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://twitter.com/bhuva_dd data-toggle=tooltip data-placement=bottom title="Follow me on Twitter" target=_blank rel=noopener aria-label="Follow me on Twitter"><i class="fab fa-twitter" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class="universal-wrapper pt-3"><h1>molecular phenotypes</h1></div><div class=universal-wrapper><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/softwares/msigdb/>msigdb</a></div><a href=/softwares/msigdb/ class=summary-link><div class=article-style>This package provides the Molecular Signatures Database (MSigDB) in a R accessible objects. Signatures are stored in GeneSet class objects form the GSEABase package and the entire database is stored in a GeneSetCollection object. These data are then hosted on the ExperimentHub. Data used in this package was obtained from the MSigDB of the Broad Institute. Metadata for each gene set is stored along with the gene set in the GeneSet class object.</div></a><div class="stream-meta article-metadata"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Gordon K Smyth</span>, <span>Alexandra Garnham</span></div><span class=article-date>Oct 15, 2022</span></div></div><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/msigdb/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/msigdb target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.msigdb target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/msigdb/issues target=_blank rel=noopener>Report a bug</a></div></div><div class=ml-3></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/softwares/singscore/>singscore</a></div><a href=/softwares/singscore/ class=summary-link><div class=article-style>A simple single-sample gene signature scoring method that uses rank-based statistics to analyze the sample&rsquo;s gene expression profile. It scores the expression activities of gene sets at a single-sample level.</div></a><div class="stream-meta article-metadata"><div class=article-metadata><div><span>Ruqian Lyu</span>, <span>Momeneh Foroutan</span>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Malvika Kharbanda</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i></div><span class=article-date>Oct 15, 2022</span></div></div><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/singscore/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.singscore target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/singscore/ target=_blank rel=noopener>Webpage</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore/issues target=_blank rel=noopener>Report a bug</a></div></div><div class=ml-3><a href=/softwares/singscore/><img src=/softwares/singscore/featured_hu7b79064d40f6ec3cd5c6251a5155e61b_251508_150x0_resize_q75_h2_lanczos_3.webp height=173 width=150 alt=singscore loading=lazy></a></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/softwares/singscoreamlmutations/>SingscoreAMLMutations</a></div><a href=/softwares/singscoreamlmutations/ class=summary-link><div class=article-style>This workflow package shows how transcriptomic signatures can be used to infer phenotypes. The workflow begins by showing how the TCGA AML transcriptomic data can be downloaded and processed using the TCGAbiolinks packages. It then shows how samples can be scored using the singscore package and signatures from the MSigDB. Finally, the predictive capacity of scores in the context of predicting a specific mutation in AML is shown.The workflow exhibits the interplay of Bioconductor packages to achieve a gene-set level analysis.</div></a><div class="stream-meta article-metadata"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Momeneh Foroutan</span>, <span>Yi Xie</span>, <span>Ruqian Lyu</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>Oct 15, 2022</span></div></div><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/singscoreamlmutations/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/SingscoreAMLMutations target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.SingscoreAMLMutations target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/SingscoreAMLMutations/issues target=_blank rel=noopener>Report a bug</a></div></div><div class=ml-3><a href=/softwares/singscoreamlmutations/><img src=/softwares/singscoreamlmutations/featured_huab255f603220787dd7053be139cd5651_93635_150x0_resize_q75_h2_lanczos_3.webp height=182 width=150 alt=SingscoreAMLMutations loading=lazy></a></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/softwares/visse/>vissE</a></div><a href=/softwares/visse/ class=summary-link><div class=article-style>This package enables the interpretation and analysis of results from a gene set enrichment analysis using network-based and text-mining approaches. Most enrichment analyses result in large lists of significant gene sets that are difficult to interpret. Tools in this package help build a similarity-based network of significant gene sets from a gene set enrichment analysis that can then be investigated for their biological function using text-mining approaches.</div></a><div class="stream-meta article-metadata"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title=maintainer></i>, <span>Ahmed Mohamed</span></div><span class=article-date>Oct 15, 2022</span></div></div><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/softwares/visse/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.18129/B9.bioc.vissE target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/vissE/ target=_blank rel=noopener>Webpage</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow target=_blank rel=noopener>Workshop</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE/issues target=_blank rel=noopener>Report a bug</a></div></div><div class=ml-3><a href=/softwares/visse/><img src=/softwares/visse/featured_hubb7914728c178d6e3f9604d4a552d94b_888199_150x0_resize_q75_h2_lanczos_3.webp height=173 width=150 alt=vissE loading=lazy></a></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/network-analysis-of-differentially-expressed-genes-using-visse-and-cytoscape/>Network analysis of differentially-expressed genes using vissE and Cytoscape</a></div><a href=/talk/network-analysis-of-differentially-expressed-genes-using-visse-and-cytoscape/ class=summary-link><div class=article-style>This workshop presents our vissE tool and shows how protein-protein interaction networks derived from it can be further interpreted using the Cytoscape network analysis tool.</div></a><div class="stream-meta article-metadata"><div><span>Sep 29, 2022</span>
<span class=middot-divider></span>
<span>Melbourne Bioinformatics</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Ahmed Mohamed</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/publication/2022_berthelet_drugpred/>Computational Screening of Anti-Cancer Drugs Identifies a New BRCA Independent Gene Expression Signature to Predict Breast Cancer Sensitivity to Cisplatin</a></div><a href=/publication/2022_berthelet_drugpred/ class=summary-link><div class=article-style>In this work, we developed models to predict drug response in cancer patients using cell line models</div></a><div class="stream-meta article-metadata"><div><span>Jean Berthelet</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Momeneh Foroutan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Holly J Whitfield</span>, <span>Farrah El-Saafin</span>, <span>Joseph Cursons</span>, <span>Antonin Serrano</span>, <span>Michal Merdas</span>, <span>Elgene Lim</span>, <span>Emmanuelle Charafe-Jauffret</span>, <span>Christophe Ginestier</span>, <span>Matthias Ernst</span>, <span>Frédéric Hollande</span>, <span>Robin L Anderson</span>, <span>Bhupinder Pal</span>, <span>Belinda Yeo</span>, <span>Melissa J Davis</span>, <span>Delphine Merino</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.mdpi.com/2072-6694/14/10/2404/pdf?version=1652425461" target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_berthelet_drugpred/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.3390/cancers14102404 target=_blank rel=noopener>DOI</a></div></div><div class=ml-3><a href=/publication/2022_berthelet_drugpred/><img src=/publication/2022_berthelet_drugpred/featured_hu6a11062e525c7ec9444e8fa867a1fedb_607034_150x0_resize_q75_h2_lanczos_3.webp height=136 width=150 alt="Computational Screening of Anti-Cancer Drugs Identifies a New BRCA Independent Gene Expression Signature to Predict Breast Cancer Sensitivity to Cisplatin" loading=lazy></a></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/publication/2022_bhuva_visse/>vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis</a></div><a href=/publication/2022_bhuva_visse/ class=summary-link><div class=article-style>vissE is a tool that can help identify, visualise and explore higher-order biological processes from the results of a gene-set enrichment analysis (GSEA)</div></a><div class="stream-meta article-metadata"><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Ning Liu</span>, <span>Holly J Whitfield</span>, <span>Nicholas Papachristos</span>, <span>Sam Lee</span>, <span>Malvika Kharbanda</span>, <span>Ahmed Mohamed</span>, <span>Melissa J Davis</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.biorxiv.org/content/10.1101/2022.03.06.483195v1.full.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_bhuva_visse/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1101/2022.03.06.483195 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://vissE.cloud target=_blank rel=noopener>Web tool</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/vissE.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow/ target=_blank rel=noopener>Workshop</a></div></div><div class=ml-3><a href=/publication/2022_bhuva_visse/><img src=/publication/2022_bhuva_visse/featured_hu41b3d5fe265c16b05a9161fd859c3f8a_857746_150x0_resize_q75_h2_lanczos_3.webp height=131 width=150 alt="vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis" loading=lazy></a></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/network-based-functional-analysis-in-transcriptomics/>Network-based functional analysis in transcriptomics</a></div><a href=/talk/network-based-functional-analysis-in-transcriptomics/ class=summary-link><div class=article-style>A workshop on using vissE to analyse transcriptomic data</div></a><div class="stream-meta article-metadata"><div><span>Nov 30, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/visse-visualising-higher-order-biological-processes-in-cancer-omics-datasets/>vissE – Visualising higher-order biological processes in cancer ‘omics datasets</a></div><a href=/talk/visse-visualising-higher-order-biological-processes-in-cancer-omics-datasets/ class=summary-link><div class=article-style>A talk on using vissE to analyse transcriptomic data</div></a><div class="stream-meta article-metadata"><div><span>Oct 8, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/talk/an-introduction-to-biological-network-analysis/>An introduction to biological network analysis</a></div><a href=/talk/an-introduction-to-biological-network-analysis/ class=summary-link><div class=article-style>This masterclass includes an introductory lecture on gene-set enrichment analysis followed by a demonstration of the vissE and singscore tools we developed.</div></a><div class="stream-meta article-metadata"><div><span>Aug 27, 2021 12:00 AM</span>
<span class=middot-divider></span>
<span>Online (South-east Asia region)</span></div><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Jinjin Chen</span>, <span>Yimin Zhang</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter mr-1"></i>Follow</a></div></div><div class=ml-3></div></div><nav class=mt-1><ul class="pagination justify-content-center"><li class=page-item><a class=page-link href=/tag/molecular-phenotypes/page/2/>&#187;</a></li></ul></nav></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2023, Dharmesh D Bhuva. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.d26509351aa0ff874abbee824e982e9b.js></script>
<script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":true}</script><script src=/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js type=module></script>
<script src=/en/js/wowchemy.min.e8ee06ba8371980ffde659871dd593b0.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>