---
title: "Profiling of lung SARS-CoV-2 and influenza virus infection dissects virus-specific host responses and gene signatures"
authors:
- Arutha Kulasinghe
- Chin Wee Tan
- Anna Flavia Ribeiro dos Santos Miggiolaro
- James Monkman
- Habib SadeghiRad
- admin
- Jarbas da Silva Motta Junior
- Caroline Busatta Vaz de Paula
- Seigo Nagashima
- Cristina Pellegrino Baena
- Paulo Souza-Fonseca-Guimaraes
- Lucia de Noronha
- Timothy McCulloch
- Gustavo Rodrigues Rossi
- Caroline Cooper
- Benjamin Tang
- Kirsty R Short
- Melissa J Davis
- Fernando Souza-Fonseca-Guimaraes
- Gabrielle T Belz
- Ken O'Byrne
date: "2022-06-02"
doi: "https://doi.org/10.1183/13993003.01881-2021"

author_notes:
  - 'Co-first author'
  - 'Co-first author'
  - 'Co-first author'

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "European Respiratory Journal"
publication_short: "ERJ"

abstract: "Background: The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which emerged in late 2019 has spread globally, causing a pandemic of respiratory illness designated coronavirus disease 2019 (COVID-19). A better definition of the pulmonary host response to SARS-CoV-2 infection is required to understand viral pathogenesis and to validate putative COVID-19 biomarkers that have been proposed in clinical studies.

Methods: Here, we use targeted transcriptomics of formalin-fixed paraffin-embedded tissue using the NanoString GeoMX platform to generate an in-depth picture of the pulmonary transcriptional landscape of COVID-19, pandemic H1N1 influenza and uninfected control patients.

Results: Host transcriptomics showed a significant upregulation of genes associated with inflammation, type I interferon production, coagulation and angiogenesis in the lungs of COVID-19 patients compared to non-infected controls. SARS-CoV-2 was non-uniformly distributed in lungs (emphasising the advantages of spatial transcriptomics) with the areas of high viral load associated with an increased type I interferon response. Once the dominant cell type present in the sample, within patient correlations and patientâ€“patient variation, had been controlled for, only a very limited number of genes were differentially expressed between the lungs of fatal influenza and COVID-19 patients. Strikingly, the interferon-associated gene IFI27, previously identified as a useful blood biomarker to differentiate bacterial and viral lung infections, was significantly upregulated in the lungs of COVID-19 patients compared to patients with influenza.

Conclusion: Collectively, these data demonstrate that spatial transcriptomics is a powerful tool to identify novel gene signatures within tissues, offering new insights into the pathogenesis of SARS-COV-2 to aid in patient triage and treatment."

# Summary. An optional shortened abstract.
summary: ""

tags:
- spatial transcriptomics
- COVID
- gene-set enrichment analysis
featured: false

links:
- name: Bioconductor
  url: https://www.bioconductor.org/packages/release/bioc/html/standR.html
url_pdf: 'https://erj.ersjournals.com/content/erj/59/6/2101881.full.pdf'
url_code: 'https://github.com/DavisLaboratory/standR'
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Spatial profiling of COVID-19 infected lung using the NanoString GeoMx platform'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
