<!doctype html><!-- This site was created with Wowchemy. https://www.wowchemy.com --><!-- Last Published: October 16, 2022 --><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/vendor-bundle.min.16f785cdb553c8c4431db6775122af35.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.dc93abdfc6ebac00d36cb8086608717e.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Dharmesh D Bhuva"><meta name=description content="A highly-customizable Hugo academic resume theme powered by Wowchemy website builder."><link rel=alternate hreflang=en-us href=https://bhuvad.github.io/publication/><link rel=canonical href=https://bhuvad.github.io/publication/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#4caf50"><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@wowchemy"><meta property="twitter:creator" content="@wowchemy"><meta property="twitter:image" content="https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_3.png"><meta property="og:site_name" content="bhuvad"><meta property="og:url" content="https://bhuvad.github.io/publication/"><meta property="og:title" content="Publications | bhuvad"><meta property="og:description" content="A highly-customizable Hugo academic resume theme powered by Wowchemy website builder."><meta property="og:image" content="https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2022-09-15T00:00:00+00:00"><link rel=alternate href=/publication/index.xml type=application/rss+xml title=bhuvad><title>Publications | bhuvad</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=3a079e7dad19be978a318345a7749d34><script src=/js/wowchemy-init.min.b6a114893adb99eba63208f045d5db5e.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>bhuvad</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>bhuvad</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#softwares><span>Softwares</span></a></li><li class=nav-item><a class=nav-link href=/#talks><span>Talks</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li><li class=nav-item><a class=nav-link href=/uploads/resume.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://twitter.com/bhuva_dd data-toggle=tooltip data-placement=bottom title="Follow me on Twitter" target=_blank rel=noopener aria-label="Follow me on Twitter"><i class="fab fa-twitter" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class="universal-wrapper pt-3"><h1>Publications</h1></div><div class=universal-wrapper><div class=row><div class=col-lg-12><div class="form-row mb-4"><div class=col-auto><input type=search class="filter-search form-control form-control-sm" placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off role=textbox spellcheck=false></div><div class=col-auto><select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group=pubtype><option value=*>Type</option><option value=.pubtype-2>
Journal article</option><option value=.pubtype-3>
Preprint</option></select></div><div class=col-auto><select class="pub-filters form-control form-control-sm" data-filter-group=year><option value=*>Date</option><option value=.year-2022>
2022</option><option value=.year-2021>
2021</option><option value=.year-2020>
2020</option><option value=.year-2019>
2019</option><option value=.year-2018>
2018</option></select></div></div><div id=container-publications><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022"><div class="card-simple view-card"><div class=article-metadata><div><span>Arutha Kulasinghe</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>NingLiu</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Chin WeeTan</span>, <span>James Monkman</span>, <span>Jane E Sinclair</span>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>David Godbolt</span>, <span>Liuliu Pan</span>, <span>Andy Nam</span>, <span>Habib Sadeghirad</span>, <span>Kei Sato</span>, <span>Gianluigi Li Bassi</span>, <span>Ken O'Byrne</span>, <span>Camila Hartmann</span>, <span>Anna Flavia Ribeiro dos Santos Miggiolaro</span>, <span>Gustavo Lenci Marques</span>, <span>Lidia ZytynskiMoura</span>, <span>Derek Richard</span>, <span>Mark Adams</span>, <span>Lucia de Noronha</span>, <span>Cristina Pellegrino Baena</span>, <span>Jacky Y Suen</span>, <span>Rakesh Arora</span>, <span>Gabrielle T Belz</span>, <span>Kirsty R Short</span>, <span>Melissa J Davis</span>, <span>Fernando Souza-Fonseca Guimaraes</span>, <span>John F Fraser</span></div><span class=article-date>September, 2022</span>
<span class=middot-divider></span>
<span class=pub-publication>Immunology</span></div><a href=/publication/2022_kulasinghe_covidheart/><div class=img-hover-zoom><img src=/publication/2022_kulasinghe_covidheart/featured_hu7e3546ba36bf3bd489dec8ae1cdeb6ea_2118681_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Transcriptomic profiling of cardiac tissues from SARS-CoV-2 patients identifies DNA damage" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2022_kulasinghe_covidheart/>Transcriptomic profiling of cardiac tissues from SARS-CoV-2 patients identifies DNA damage</a></div><a href=/publication/2022_kulasinghe_covidheart/ class=summary-link><div class=article-style><p>Abstract The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is known to present with pulmonary and extra-pulmonary organ complications. In comparison with the 2009 pandemic (pH1N1), SARS-CoV-2 infection is likely to lead to more severe disease, with multi-organ effects, including cardiovascular disease. SARS-CoV-2 has been associated with acute and long-term cardiovascular disease, but the molecular changes that govern this remain unknown. In this study, we investigated the host transcriptome landscape of cardiac tissues collected at rapid autopsy from seven SARS-CoV-2, two pH1N1, and six control patients using targeted spatial transcriptomics approaches. Although SARS-CoV-2 was not detected in cardiac tissue, host transcriptomics showed upregulation of genes associated with DNA damage and repair, heat shock, and M1-like macrophage infiltration in the cardiac tissues of COVID-19 patients. The DNA damage present in the SARS-CoV-2 patient samples, were further confirmed by γ-H2Ax immunohistochemistry. In comparison, pH1N1 showed upregulation of interferon-stimulated genes, in particular interferon and complement pathways, when compared with COVID-19 patients. These data demonstrate the emergence of distinct transcriptomic profiles in cardiac tissues of SARS-CoV-2 and pH1N1 influenza infection supporting the need for a greater understanding of the effects on extra-pulmonary organs, including the cardiovascular system of COVID-19 patients, to delineate the immunopathobiology of SARS-CoV-2 infection, and long term impact on health.</p></div></a><div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_kulasinghe_covidheart/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/standR target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1111/imm.13577 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/standR.html target=_blank rel=noopener>Bioconductor</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022"><div class="card-simple view-card"><div class=article-metadata><div><span>Arutha Kulasinghe</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Chin Wee Tan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Anna Flavia Ribeiro dos Santos Miggiolaro</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>James Monkman</span>, <span>Habib Sadeghirad</span>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Jarbas da Silva Motta Junior</span>, <span>Caroline Busatta Vaz de Paula</span>, <span>Seigo Nagashima</span>, <span>Cristina Pellegrino Baena</span>, <span>Paulo Souza-Fonseca-Guimaraes</span>, <span>Lucia de Noronha</span>, <span>Timothy McCulloch</span>, <span>Gustavo Rodrigues Rossi</span>, <span>Caroline Cooper</span>, <span>Benjamin Tang</span>, <span>Kirsty R Short</span>, <span>Melissa J Davis</span>, <span>Fernando Souza-Fonseca Guimaraes</span>, <span>Gabrielle T Belz</span>, <span>Ken O'Byrne</span></div><span class=article-date>June, 2022</span>
<span class=middot-divider></span>
<span class=pub-publication>ERJ</span></div><a href=/publication/2022_kulasinghe_covidlung/><div class=img-hover-zoom><img src=/publication/2022_kulasinghe_covidlung/featured_hu4620166425835ed2b4948100ad9e2335_307619_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Profiling of lung SARS-CoV-2 and influenza virus infection dissects virus-specific host responses and gene signatures" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2022_kulasinghe_covidlung/>Profiling of lung SARS-CoV-2 and influenza virus infection dissects virus-specific host responses and gene signatures</a></div><a href=/publication/2022_kulasinghe_covidlung/ class=summary-link><div class=article-style><p>Background: The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which emerged in late 2019 has spread globally, causing a pandemic of respiratory illness designated coronavirus disease 2019 (COVID-19). A better definition of the pulmonary host response to SARS-CoV-2 infection is required to understand viral pathogenesis and to validate putative COVID-19 biomarkers that have been proposed in clinical studies.
Methods: Here, we use targeted transcriptomics of formalin-fixed paraffin-embedded tissue using the NanoString GeoMX platform to generate an in-depth picture of the pulmonary transcriptional landscape of COVID-19, pandemic H1N1 influenza and uninfected control patients.
Results: Host transcriptomics showed a significant upregulation of genes associated with inflammation, type I interferon production, coagulation and angiogenesis in the lungs of COVID-19 patients compared to non-infected controls. SARS-CoV-2 was non-uniformly distributed in lungs (emphasising the advantages of spatial transcriptomics) with the areas of high viral load associated with an increased type I interferon response. Once the dominant cell type present in the sample, within patient correlations and patient–patient variation, had been controlled for, only a very limited number of genes were differentially expressed between the lungs of fatal influenza and COVID-19 patients. Strikingly, the interferon-associated gene IFI27, previously identified as a useful blood biomarker to differentiate bacterial and viral lung infections, was significantly upregulated in the lungs of COVID-19 patients compared to patients with influenza.
Conclusion: Collectively, these data demonstrate that spatial transcriptomics is a powerful tool to identify novel gene signatures within tissues, offering new insights into the pathogenesis of SARS-COV-2 to aid in patient triage and treatment.</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://erj.ersjournals.com/content/erj/59/6/2101881.full.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_kulasinghe_covidlung/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/standR target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1183/13993003.01881-2021 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/standR.html target=_blank rel=noopener>Bioconductor</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022"><div class="card-simple view-card"><div class=article-metadata><div><span>Jean Berthelet</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Momeneh Foroutan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Holly J Whitfield</span>, <span>Farrah El-Saafin</span>, <span>Joseph Cursons</span>, <span>Antonin Serrano</span>, <span>Michal Merdas</span>, <span>Elgene Lim</span>, <span>Emmanuelle Charafe-Jauffret</span>, <span>Christophe Ginestier</span>, <span>Matthias Ernst</span>, <span>Frédéric Hollande</span>, <span>Robin L Anderson</span>, <span>Bhupinder Pal</span>, <span>Belinda Yeo</span>, <span>Melissa J Davis</span>, <span>Delphine Merino</span></div><span class=article-date>May, 2022</span>
<span class=middot-divider></span>
<span class=pub-publication>Cancers</span></div><a href=/publication/2022_berthelet_drugpred/><div class=img-hover-zoom><img src=/publication/2022_berthelet_drugpred/featured_hu6a11062e525c7ec9444e8fa867a1fedb_607034_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Computational Screening of Anti-Cancer Drugs Identifies a New BRCA Independent Gene Expression Signature to Predict Breast Cancer Sensitivity to Cisplatin" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2022_berthelet_drugpred/>Computational Screening of Anti-Cancer Drugs Identifies a New BRCA Independent Gene Expression Signature to Predict Breast Cancer Sensitivity to Cisplatin</a></div><a href=/publication/2022_berthelet_drugpred/ class=summary-link><div class=article-style><p>In this work, we developed models to predict drug response in cancer patients using cell line models</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.mdpi.com/2072-6694/14/10/2404/pdf?version=1652425461" target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_berthelet_drugpred/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.3390/cancers14102404 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2022"><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Chin Wee Tan</span>, <span>Ning Liu</span>, <span>Holly J Whitfield</span>, <span>Nicholas Papachristos</span>, <span>Sam Lee</span>, <span>Malvika Kharbanda</span>, <span>Ahmed Mohamed</span>, <span>Melissa J Davis</span></div><span class=article-date>March, 2022</span>
<span class=middot-divider></span>
<span class=pub-publication></span></div><a href=/publication/2022_bhuva_visse/><div class=img-hover-zoom><img src=/publication/2022_bhuva_visse/featured_hu41b3d5fe265c16b05a9161fd859c3f8a_857746_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2022_bhuva_visse/>vissE: A versatile tool to identify and visualise higher-order molecular phenotypes from functional enrichment analysis</a></div><a href=/publication/2022_bhuva_visse/ class=summary-link><div class=article-style><p>vissE is a tool that can help identify, visualise and explore higher-order biological processes from the results of a gene-set enrichment analysis (GSEA)</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.biorxiv.org/content/10.1101/2022.03.06.483195v1.full.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2022_bhuva_visse/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/vissE target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1101/2022.03.06.483195 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://vissE.cloud target=_blank rel=noopener>Web tool</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/vissE.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow/ target=_blank rel=noopener>Workshop</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021"><div class="card-simple view-card"><div class=article-metadata><div><span>Laura Genovesi</span>, <span>Amanda Millar</span>, <span>Elissa Tolson</span>, <span>Matthew Singleton</span>, <span>Emily Hassall</span>, <span>Marija Kojic</span>, <span>Caterina Brighi</span>, <span>Emily Girard</span>, <span>Clara Andradas</span>, <span>Mani Kuchibhotla</span>, <span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Raelene Endersby</span>, <span>Nicholas G Gottardo</span>, <span>Anne Bernard</span>, <span>Christelle Adolphe</span>, <span>James M Olson</span>, <span>Melissa J Davis</span>, <span>Brandon J Wainwright</span></div><span class=article-date>June, 2021</span>
<span class=middot-divider></span>
<span class=pub-publication>Genome Med</span></div><a href=/publication/2021_genovesi_mb/><div class=img-hover-zoom><img src=/publication/2021_genovesi_mb/featured_hu5c8c907b4fad729599c24ca713a35b54_1825169_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Systems pharmacogenomics identifies novel targets and clinically actionable therapeutics for medulloblastoma" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2021_genovesi_mb/>Systems pharmacogenomics identifies novel targets and clinically actionable therapeutics for medulloblastoma</a></div><a href=/publication/2021_genovesi_mb/ class=summary-link><div class=article-style><p>Background: Medulloblastoma (MB) is the most common malignant paediatric brain tumour and a leading cause of cancer-related mortality and morbidity. Existing treatment protocols are aggressive in nature resulting in significant neurological, intellectual and physical disabilities for the children undergoing treatment. Thus, there is an urgent need for improved, targeted therapies that minimize these harmful side effects.
Methods: We identified candidate drugs for MB using a network-based systems-pharmacogenomics approach: based on results from a functional genomics screen, we identified a network of interactions implicated in human MB growth regulation. We then integrated drugs and their known mechanisms of action, along with gene expression data from a large collection of medulloblastoma patients to identify drugs with potential to treat MB.
Results: Our analyses identified drugs targeting CDK4, CDK6 and AURKA as strong candidates for MB; all of these genes are well validated as drug targets in other tumour types. We also identified non-WNT MB as a novel indication for drugs targeting TUBB, CAD, SNRPA, SLC1A5, PTPRS, P4HB and CHEK2. Based upon these analyses, we subsequently demonstrated that one of these drugs, the new microtubule stabilizing agent, ixabepilone, blocked tumour growth in vivo in mice bearing patient-derived xenograft tumours of the Sonic Hedgehog and Group 3 subtype, providing the first demonstration of its efficacy in MB.
Conclusions: Our findings confirm that this data-driven systems pharmacogenomics strategy is a powerful approach for the discovery and validation of novel therapeutic candidates relevant to MB treatment, and along with data validating ixabepilone in PDX models of the two most aggressive subtypes of medulloblastoma, we present the network analysis framework as a resource for the field.</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://genomemedicine.biomedcentral.com/counter/pdf/10.1186/s13073-021-00920-z.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2021_genovesi_mb/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1186/s13073-021-00920-z target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>November, 2020</span>
<span class=middot-divider></span>
<span class=pub-publication>NAR</span></div><a href=/publication/2020_bhuva_stingscore/><div class=img-hover-zoom><img src=/publication/2020_bhuva_stingscore/featured_hub537821e4a633d9e4cc6e4475cf45b6e_102410_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Stable gene expression for normalisation and single-sample scoring" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2020_bhuva_stingscore/>Stable gene expression for normalisation and single-sample scoring</a></div><a href=/publication/2020_bhuva_stingscore/ class=summary-link><div class=article-style><p>This work proposes an extension to our robust single-sample molecular phenotyping method, singscore, that works on targeted transcriptomic panels with as few as 5 endogenous control genes, thus taking the approach a step closer to the clinic.</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://academic.oup.com/nar/article-pdf/48/19/e113/34133573/gkaa802.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2020_bhuva_stingscore/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1093/nar/gkaa802 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/singscore.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow/ target=_blank rel=noopener>Workshop</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Joseph Cursons</span>, <span>Gordon K Smyth</span>, <span>Melissa J Davis</span></div><span class=article-date>November, 2019</span>
<span class=middot-divider></span>
<span class=pub-publication>Genome Biol.</span></div><a href=/publication/2019_bhuva_dcanr/><div class=img-hover-zoom><img src=/publication/2019_bhuva_dcanr/featured_hu86d665dad16700006f56a8957fd529b1_656008_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Differential co-expression-based detection of conditional relationships in transcriptional data: comparative analysis and application to breast cancer" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2019_bhuva_dcanr/>Differential co-expression-based detection of conditional relationships in transcriptional data: comparative analysis and application to breast cancer</a></div><a href=/publication/2019_bhuva_dcanr/ class=summary-link><div class=article-style><p>dcanr is a method that helps infer differential regulatory networks from high-througput &lsquo;omics data</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://genomebiology.biomedcentral.com/counter/pdf/10.1186/s13059-019-1851-8.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2019_bhuva_dcanr/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/dcanr target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.26188/5cb8049f593a6 target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1186/s13059-019-1851-8 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/dcanr.html target=_blank rel=noopener>Bioconductor</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class="card-simple view-card"><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Momeneh Foroutan</span>, <span>Yi Xie</span>, <span>Ruqian Lyu</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>October, 2019</span>
<span class=middot-divider></span>
<span class=pub-publication>F1000Research</span></div><a href=/publication/2019_bhuva_singscoreamlmutations/><div class=img-hover-zoom><img src=/publication/2019_bhuva_singscoreamlmutations/featured_huab255f603220787dd7053be139cd5651_93635_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Using singscore to predict mutation status in acute myeloid leukemia from transcriptomic signatures" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2019_bhuva_singscoreamlmutations/>Using singscore to predict mutation status in acute myeloid leukemia from transcriptomic signatures</a></div><a href=/publication/2019_bhuva_singscoreamlmutations/ class=summary-link><div class=article-style><p>This workflow demonstrates the versatility of singscore by applying it to infer mutation status in cancer</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://f1000research.com/articles/8-776/v3/pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2019_bhuva_singscoreamlmutations/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.12688/f1000research.19236.3 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018"><div class="card-simple view-card"><div class=article-metadata><div><span>Momeneh Foroutan</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span class=author-highlighted>Dharmesh D Bhuva</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Co-first author"></i>, <span>Ruqian Lyu</span>, <span>Kristy Horan</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>November, 2018</span>
<span class=middot-divider></span>
<span class=pub-publication>BMC Bioinform</span></div><a href=/publication/2018_foroutan_singscore/><div class=img-hover-zoom><img src=/publication/2018_foroutan_singscore/featured_hub771a9a426745829c866a5131a2ff602_79933_808x455_fill_q75_h2_lanczos_smart1_3.webp height=455 width=808 class=article-banner alt="Single sample scoring of molecular phenotypes" loading=lazy></div></a><div class="section-subheading article-title mb-1 mt-3"><a href=/publication/2018_foroutan_singscore/>Single sample scoring of molecular phenotypes</a></div><a href=/publication/2018_foroutan_singscore/ class=summary-link><div class=article-style><p>singscore is a robust rank-based method to compute molecular phenotypes scores for individual samples such as those originating from an individual patient in the clinic</p></div></a><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bmcbioinformatics.biomedcentral.com/counter/pdf/10.1186/s12859-018-2435-4.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/2018_foroutan_singscore/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.youtube.com/watch?v=pyHG9Qww-xk" target=_blank rel=noopener>Video</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/https://doi.org/10.1186/s12859-018-2435-4 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.bioconductor.org/packages/release/bioc/html/singscore.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://bioconductor.org/packages/release/workflows/html/SingscoreAMLMutations.html target=_blank rel=noopener>Workshop</a></div></div></div></div></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2022, Dharmesh D Bhuva. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.d26509351aa0ff874abbee824e982e9b.js></script>
<script src=https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":true}</script><script src=/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js type=module></script>
<script src=/en/js/wowchemy.min.e8ee06ba8371980ffde659871dd593b0.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>