<!doctype html><!-- This site was created with Wowchemy. https://www.wowchemy.com --><!-- Last Published: May 16, 2023 --><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/vendor-bundle.min.16f785cdb553c8c4431db6775122af35.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.dc93abdfc6ebac00d36cb8086608717e.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Dharmesh D Bhuva"><meta name=description content="This work proposes an extension to our robust single-sample molecular phenotyping method, singscore, that works on targeted transcriptomic panels with as few as 5 endogenous control genes, thus taking the approach a step closer to the clinic."><link rel=alternate hreflang=en-us href=https://bhuvad.github.io/publication/2020_bhuva_stingscore/><link rel=canonical href=https://bhuvad.github.io/publication/2020_bhuva_stingscore/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#4caf50"><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@wowchemy"><meta property="twitter:creator" content="@wowchemy"><meta property="twitter:image" content="https://bhuvad.github.io/publication/2020_bhuva_stingscore/featured.png"><meta property="og:site_name" content="bhuvad"><meta property="og:url" content="https://bhuvad.github.io/publication/2020_bhuva_stingscore/"><meta property="og:title" content="Stable gene expression for normalisation and single-sample scoring | bhuvad"><meta property="og:description" content="This work proposes an extension to our robust single-sample molecular phenotyping method, singscore, that works on targeted transcriptomic panels with as few as 5 endogenous control genes, thus taking the approach a step closer to the clinic."><meta property="og:image" content="https://bhuvad.github.io/publication/2020_bhuva_stingscore/featured.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2020-11-04T00:00:00+00:00"><meta property="article:modified_time" content="2020-11-04T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://bhuvad.github.io/publication/2020_bhuva_stingscore/"},"headline":"Stable gene expression for normalisation and single-sample scoring","image":["https://bhuvad.github.io/publication/2020_bhuva_stingscore/featured.png"],"datePublished":"2020-11-04T00:00:00Z","dateModified":"2020-11-04T00:00:00Z","author":{"@type":"Person","name":"Dharmesh D Bhuva"},"publisher":{"@type":"Organization","name":"bhuvad","logo":{"@type":"ImageObject","url":"https://bhuvad.github.io/media/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_3.png"}},"description":"This work proposes an extension to our robust single-sample molecular phenotyping method, singscore, that works on targeted transcriptomic panels with as few as 5 endogenous control genes, thus taking the approach a step closer to the clinic."}</script><title>Stable gene expression for normalisation and single-sample scoring | bhuvad</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=a67cff4898576e36f2aee3eccd8d733f><script src=/js/wowchemy-init.min.b6a114893adb99eba63208f045d5db5e.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>bhuvad</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>bhuvad</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#softwares><span>Softwares</span></a></li><li class=nav-item><a class=nav-link href=/#talks><span>Talks</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li><li class=nav-item><a class=nav-link href=/uploads/resume.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://twitter.com/bhuva_dd data-toggle=tooltip data-placement=bottom title="Follow me on Twitter" target=_blank rel=noopener aria-label="Follow me on Twitter"><i class="fab fa-twitter" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class=pub><div class="article-container pt-3"><h1>Stable gene expression for normalisation and single-sample scoring</h1><div class=article-metadata><div><span class=author-highlighted>Dharmesh D Bhuva</span>, <span>Joseph Cursons</span>, <span>Melissa J Davis</span></div><span class=article-date>November, 2020</span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary btn-page-header" href=https://academic.oup.com/nar/article-pdf/48/19/e113/34133573/gkaa802.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header js-cite-modal" data-filename=/publication/2020_bhuva_stingscore/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header" href=https://github.com/DavisLaboratory/singscore target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header" href=https://doi.org/https://doi.org/10.1093/nar/gkaa802 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header" href=https://www.bioconductor.org/packages/release/bioc/html/singscore.html target=_blank rel=noopener>Bioconductor</a>
<a class="btn btn-outline-primary btn-page-header" href=https://davislaboratory.github.io/GenesetAnalysisWorkflow/ target=_blank rel=noopener>Workshop</a></div></div><div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:720px;max-height:334px><div style=position:relative><img src=/publication/2020_bhuva_stingscore/featured_hub537821e4a633d9e4cc6e4475cf45b6e_102410_720x2500_fit_q75_h2_lanczos_3.webp width=720 height=334 alt class=featured-image>
<span class=article-header-caption>Schematic of stingscore (singscore with stably expressed genes)</span></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract>Gene expression signatures have been critical in defining the molecular phenotypes of cells, tissues, and patient samples. Their most notable and widespread clinical application is stratification of breast cancer patients into molecular (PAM50) subtypes. The cost and relatively large amounts of fresh starting material required for whole-transcriptome sequencing has limited clinical application of thousands of existing gene signatures captured in repositories such as the Molecular Signature Database. We identified genes with stable expression across a range of abundances, and with a preserved relative ordering across thousands of samples, allowing signature scoring and supporting general data normalisation for transcriptomic data. Our new method, stingscore, quantifies and summarises relative expression levels of signature genes from individual samples through the inclusion of these ‘stably-expressed genes’. We show that our list of stable genes has better stability across cancer and normal tissue data than previously proposed gene sets. Additionally, we show that signature scores computed from targeted transcript measurements using stingscore can predict docetaxel response in breast cancer patients. This new approach to gene expression signature analysis will facilitate the development of panel-type tests for gene expression signatures, thus supporting clinical translation of the powerful insights gained from cancer transcriptomic studies.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#2>Journal article</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9">Nucleic Acids Research</div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style></div><div class=article-tags><a class="badge badge-light" href=/tag/gene-set-enrichment-analysis/>gene-set enrichment analysis</a>
<a class="badge badge-light" href=/tag/molecular-phenotypes/>molecular phenotypes</a>
<a class="badge badge-light" href=/tag/stably-expressed-genes/>stably expressed genes</a>
<a class="badge badge-light" href=/tag/endogenous-control-rna/>endogenous control RNA</a>
<a class="badge badge-light" href=/tag/breast-cancer/>breast cancer</a></div><div class=share-box><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fbhuvad.github.io%2Fpublication%2F2020_bhuva_stingscore%2F&amp;text=Stable+gene+expression+for+normalisation+and+single-sample+scoring" target=_blank rel=noopener class=share-btn-twitter aria-label=twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https%3A%2F%2Fbhuvad.github.io%2Fpublication%2F2020_bhuva_stingscore%2F&amp;t=Stable+gene+expression+for+normalisation+and+single-sample+scoring" target=_blank rel=noopener class=share-btn-facebook aria-label=facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=Stable%20gene%20expression%20for%20normalisation%20and%20single-sample%20scoring&amp;body=https%3A%2F%2Fbhuvad.github.io%2Fpublication%2F2020_bhuva_stingscore%2F" target=_blank rel=noopener class=share-btn-email aria-label=envelope><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https%3A%2F%2Fbhuvad.github.io%2Fpublication%2F2020_bhuva_stingscore%2F&amp;title=Stable+gene+expression+for+normalisation+and+single-sample+scoring" target=_blank rel=noopener class=share-btn-linkedin aria-label=linkedin-in><i class="fab fa-linkedin-in"></i></a></li><li><a href="whatsapp://send?text=Stable+gene+expression+for+normalisation+and+single-sample+scoring%20https%3A%2F%2Fbhuvad.github.io%2Fpublication%2F2020_bhuva_stingscore%2F" target=_blank rel=noopener class=share-btn-whatsapp aria-label=whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https%3A%2F%2Fbhuvad.github.io%2Fpublication%2F2020_bhuva_stingscore%2F&amp;title=Stable+gene+expression+for+normalisation+and+single-sample+scoring" target=_blank rel=noopener class=share-btn-weibo aria-label=weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><a href=https://bhuvad.github.io/><img class="avatar mr-3 avatar-circle" src=/authors/admin/avatar_hu5d6b5e2e7a2f38c6bfdd740fce699233_88112_270x270_fill_q75_lanczos_center.jpg alt="Dharmesh D Bhuva"></a><div class=media-body><h5 class=card-title><a href=https://bhuvad.github.io/>Dharmesh D Bhuva</a></h5><h6 class=card-subtitle>Senior post-doctoral researcher at SAiGENCI</h6><p class=card-text>My research interests include cancer systems biology, spatial statistics and computational biology.</p><ul class=network-icon aria-hidden=true><li><a href=/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/bhuva_dd target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href="https://scholar.google.com/citations?user=Jz-azvUAAAAJ&amp;hl=en" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li><li><a href=https://github.com/bhuvad target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href=https://orcid.org/0000-0002-6398-9157 target=_blank rel=noopener><i class="ai ai-orcid"></i></a></li><li><a href=https://www.linkedin.com/in/dbhuva/ target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href=/uploads/resume.pdf><i class="ai ai-cv"></i></a></li></ul></div></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2023, Dharmesh D Bhuva. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.d26509351aa0ff874abbee824e982e9b.js></script>
<script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":true}</script><script src=/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js type=module></script>
<script src=/en/js/wowchemy.min.e8ee06ba8371980ffde659871dd593b0.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>